src/assets/scripts/airport/Arrival/ArrivalBase.js

```diff
@@ -301,20 +513,21 @@ export default class ArrivalBase {
 
     /**
      * Determine delay until next spawn
+     *
+     * @for ArrivalBase
+     * @method nextInterval
+     * @return {number}
      */
     nextInterval() {
-        // TODO: these next 3 vars could easily be moved to a constants file
-        // nautical miles
-        const min_entrail = 5.5;
-        // in seconds
-        const min_interval = min_entrail * (3600 / this.speed);
-        let tgt_interval = 3600 / this.frequency;
```

Comment:
The `frequency` property is specified in acph (aircraft per hour), since that is a common (and obviously the most important) metric used in macro-level traffic management... but the game's time is all dealt with in seconds. This is just a way to translate `aircraft / hour` into `seconds / aircraft`.

Perhaps you're now realizing the magic number comes from:
`1sec * (60sec/min) * (60min/hr) ==> 3600sec/hr` 😜